LEAF_Request_Portal/js/parallelProcessing.js

```diff
@@ -46,17 +46,20 @@ function parallelProcessing(recordID, orgChartPath, CSRFToken)
         $.ajax({
             type: 'GET',
             url: 'api/?a=form/'+recordID+'/workflow/indicator/assigned',
-            success: function(obj) {
-                indicatorObject = obj;
-                for (var i = 0; i < indicatorObject.length; i++) {
-                    $(document.createElement('option'))
-                        .attr('value', indicatorObject[i].indicatorID)
-                        .html(indicatorObject[i].name)
-                        .appendTo($("select#indicator_selector"));
+            success: function(arr) {
+                indicatorObject = arr;
```

Comment:
Why is this line here?  "arr" is not used otherwise.  Why not just make the argument that's passed "indicatorObject"?

Reply:
That was Carrie and myself first thought haha but we went along with it... Definitely will change

Reply:
I had found an error log and at first I thought it was because of the loop, due to the param name (the type of an array is still object, but getting and utilizing the length property is different), so I changed the param name and where it had been used to be more specific (and overlooked that it was otherwise weird to begin with).  Will update to be more consistent.